pkg/specs/githubclient/client.go

```diff
@@ -18,7 +18,8 @@ import (
 	"github.com/google/go-github/github"
 	"github.com/pkg/errors"
 	errors2 "github.com/replicatedhq/ship/pkg/util/errors"
-	"github.com/spf13/afero"
+
+	// "github.com/spf13/afero"
```

Comment:
Should this be removed entirely?

Reply:
i removed afero from this package during the initial attempt, but ended up keeping it elsewhere, so I put it back.